src/screens/augmentedReality/ArtworkDetailScreen.js

```diff
@@ -16,28 +16,16 @@ import {
 } from '../../components';
 import { texts } from '../../config';
 import { checkDownloadedData, downloadObject, DOWNLOAD_TYPE } from '../../helpers';
-import { usePullToRefetch, useStaticContent } from '../../hooks';
 import { ScreenName } from '../../types';
 
 export const ArtworkDetailScreen = ({ route, navigation }) => {
-  const {
-    data: artworkDetail = '',
-    error,
-    loading,
-    refetch
-  } = useStaticContent({
-    type: 'html',
-    name: 'artworkDetail'
-  });
-
   const [isModalVisible, setIsModalVisible] = useState(false);
   const [data, setData] = useState(route?.params?.data ?? []);
-  const [isLoading, setIsLoading] = useState(loading);
+  const [isLoading, setIsLoading] = useState(true);
   const index = route?.params?.index;
+  const description = data?.[index]?.description || undefined;
```

Comment:
why have you added `|| undefined`. what do you think is the benefit?

Reply:
I think it can be deleted.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/669/commits/fd0b64e1f35b55330d35672b966b746520c1c8e1

Reply:
and why?

Reply:
I added it to avoid an error in case there is no description.

Reply:
`data?.[index]?.description` this results in `undefined` if there is no description, so this would be `undefined || undefiend`, you know. that's why we do not need `|| undefined`.